### PR TITLE
added cometh-matic-psp in the polygon ui

### PR DESCRIPTION
**Pull request by ongmorel**

### CODE REVIEW

File: src/features/configure/vault/polygon_pools.js [hunk 1]
```diff
@@ -33,6 +33,41 @@ export const polygonPools = [
     buyTokenUrl:
       'https://quickswap.exchange/#/swap?inputCurrency=0x7ceB23fD6bC0adD59E62ac25578270cFf1b9f619&outputCurrency=0xfbdd194376de19a88118e84e279b977f165d01b8',
   },
+  {
+    id: 'cometh-matic-psp',
+    name: 'MATIC-PSP LP',
+    token: 'MATIC-PSP LP',
+    tokenDescription: 'Cometh',
+    tokenAddress: '0xe3ae064907e9d49c7355504c25f8cae2e7831501',
```
Comment (sircryptomaxi) [inline src/features/configure/vault/polygon_pools.js:41]: @ongmorel Can you please checksum this address?

**Comment by sircryptomaxi** [comment]
@MirthFutures Contracts LGTM, only routing for lp1 could be improved to go directly from PSP->MUST

File: src/features/configure/vault/polygon_pools.js (hunk 1 above)
Comment (ongmorel) [reply src/features/configure/vault/polygon_pools.js]: done

**Comment by sircryptomaxi** [comment]
> i change the route ?
Yes, it will mean redeploying but a maintainer may have additional changes to request so hold off for the moment.

**Comment by ongmorel** [comment]
> > i change the route ?
> 
> Yes, it will mean redeploying but a maintainer may have additional changes to request so hold off for the moment.

okay keep me tuned 👍

**Pull request closed without merging**